.gitignore

```diff
@@ -20,6 +20,7 @@ MANIFEST
 pip-wheel-metadata
 .hypothesis
 doctests.py
+coverage.xml
```

Comment:
Why does `.gitignore` have to be updated?

Reply:
This is an OK addition